communication/tasks.py

```diff
@@ -19,18 +19,19 @@ def send_smses():
     fail = 0
 
     for sms in smses:
-        sent = False
         if fail < 3:
             try:
                 send_sms(sms.harambee.candidate_id, sms.message)
```

Comment:
I thought we are going to ensure this is utf-8 with `unicode(sms.message, "utf-8")` ?